Frameworks/Blockcerts/Certificate.swift

```diff
@@ -50,6 +52,43 @@ public enum CertificateParserError : Error {
 
 /// CertificateParser should never be instantiated. Call one of its `parse` methods to turn a Data object into a Certificate.
 public enum CertificateParser {
```

Comment:
Is there any reason to make it enum.

Reply:
Hmm I have no idea, this was done by another dev before me (probably better experienced too :))
A `class` would indeed make more sense there.